packages/browser/src/utils/lazyLoadIntegration.ts

```diff
@@ -68,7 +69,14 @@ export async function lazyLoadIntegration(
     script.addEventListener('error', reject);
   });
 
-  WINDOW.document.body.appendChild(script);
+  const currentScript = WINDOW.document.currentScript;
+  const parent = (currentScript && currentScript.parentElement) || WINDOW.document.body || WINDOW.document.head;
```

Comment:
l: Not 100% sure, but I think I would flip this around and do the currentScript stuff at the end rather then the beginning? so `body || head || currentScript`? 🤔 Not sure though and probably does not matter, maybe it's just because I don't really know/knew currentScript so I feel less comfortable around it 😅

Reply:
sure, no strong opinions either. Will turn it around :)